src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -224,6 +224,10 @@ const en = ({ isDivorce, partner, userCase }: CommonContent, alternativeServiceT
       link: '/downloads/bailiff-service',
     },
   },
+  subHeading1:
+    userCase.state === State.AwaitingAmendedApplication
+      ? 'Latest information'
+      : `${userCase.state === 'AwaitingClarification' ? 'What you need to do now' : 'Latest update'}`,
```

Comment:
userCase.state === 'AwaitingClarification' will never be true since it's only got to that line if userCase.state === State.AwaitingAmendedApplication

Reply:
also where is this subHeading1 used?